### DISCUSSION TITLE
Can't disable Dynamic Audio Power Management

### DISCUSSION THREAD

**Discussion by Phlipp15** [dtcooper/raspotify#513] [Q&A] [answered]
I have the newest Raspbian OS installed and Raspotify at Version 0.31.9.

I have the problem, that the sound stops after a few seconds or minutes.
I found out that the problem should be the Dynamic Audio Power Management.
If i configure the following, like described in the wiki, i cant establish a connection with Raspbian from the Spotify App.

_If you want to disable this feature, create a file called snd_soc_core.conf in /etc/modprobe.d with this line in:
options snd_soc_core pmdown_time -1_

With an older OS and older Raspbian Version it worked like a charm.

What did i do wrong?
Thanks a lot

**Comment by JasonLG1979** [comment, marked as answer]
I think it's supposed to be:

`options snd_soc_core pmdown_time=-1`?

If that works I will change the wiki.

**Reply by Phlipp15** [reply]
I tried that already and it did not work. I also tried:
options snd-soc-core pmdown_time=-1

I had installed Raspbian OS in Desktop Version. Now i installed Raspbian OS Lite and there i dont need to disable the Audio Power Management.

**Reply by JasonLG1979** [reply]
Yep, I don't know? In a pinch I've seen people run aplay in the background pushing silence forcing the device to stay awake.
